### DISCUSSION TITLE
About _thread.allocate_lock bug feedback

### DISCUSSION THREAD

**Discussion by ghluuuuuu** [RT-Thread-packages/micropython#179] [Q&A]
Repeat exec `_thread.allocate_lock` the mcu will assert crash.
----
![bug reappearance](https://oss-club.rt-thread.org/uploads/20210508/863e9c04624d966cc74b43a5ed663604.gif)

**Comment by ghluuuuuu** [comment]
@armink

**Reply by SummerGift** [reply]
上面提示了栈大小最小要大于 8k 先增大试试看。
